### PR TITLE
feat(input): step feature added for number input

### LINKED ISSUE #370: Input Component Step Feature
**Is your feature request related to a problem? Please describe.**
For input components that has `type='number'` attribute; Increase and decrease feature runs one by one with keyboard and input arrow keys. I want to Increase and decrease by specific step number instead of one by one. 
At the below gifs keyboard arrow keys used to increase and decrease. 
Baklava Playground (1)

**Describe the solution you'd like**
May be set additional step prop to input component and provide native step experience.
Baklava Playground

**Describe alternatives you've considered**
My best advice; use native step feature.

**Additional context**
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step

### PR DESCRIPTION
**Pull request by yasin749**
Closes #370

### CODE REVIEW

**Comment by yasin749** [comment]
#370

**Comment by yasin749** [comment]
#370

**Comment by yasin749** [comment]
#370

File: src/components/input/bl-input.stories.mdx [hunk 1]
```diff
@@ -85,6 +89,7 @@ export const SingleInputTemplate = (args) => html`<bl-input
     maxlength='${ifDefined(args.maxlength)}'
     min='${ifDefined(args.min)}'
     max='${ifDefined(args.max)}'
+    step='${ifDefined(args.max)}'
```
Comment (muratcorlu) [inline src/components/input/bl-input.stories.mdx:92]:
```suggestion
    step='${ifDefined(args.step)}'
```

**Review by muratcorlu — APPROVED** [review]
Hi @yasin749 Thanks for the contribution. That's a very good addition.

There is a problem in our pipeline that we hit multiple times (mentioned in #173). This time I want to fix that issue instead of walking around it. I'll create a PR to fix that issue first, then we can merge your work.

**Comment by yasin749** [comment]
Thank you @muratcorlu

**Pull request merged**